test/e2e/install/install.go

```diff
@@ -58,9 +58,8 @@ var _ = Describe("[Install] continuousIntegration", func() {
 	})
 
 	Context("with a running cluster with all the components", func() {
-		It("should perform overall deployment and verify the condition is reported as available", func() {
+		It("[test_id:47574] should perform overall deployment and verify the condition is reported as available", func() {
```

Comment:
Do you mind explaining in the PR for the novices, what is the meaning of this number?

Reply:
the test_id is the test case identifier as provided by our QE folks. We need them to match the e2e test with their test cases, so we know that if e2e test $WHATEVER passes (or not), then the testcase 12345 is satisfied (or not).